doc/15-troubleshooting.md

```diff
@@ -1,5 +1,51 @@
 # Icinga 2 Troubleshooting <a id="troubleshooting"></a>
 
+## Icinga starts/restarts/reloads very slowly
+
+### If you run Linux and systemd
+
+On RHEL or derivates add the EPEL repository first.
+Let your package manager search for package names containing "jemalloc".
+Pick preferably one named "libjemalloc" followed by a number,
+just "jemalloc" otherwise, and install it.
+
+Run `ldconfig -p |grep libjemalloc`. It should print something similar to:
+
+```
+	libjemalloc.so.2 (libc6,x86-64) => /lib64/libjemalloc.so.2
+```
+
+I.e. a relative file name followed by an absolute one. Remember the latter.
+
+Measure how long Icinga needs to load its config without and with libjemalloc:
+
+```bash
+time icinga2 daemon -C
+
+LD_PRELOAD=/lib64/libjemalloc.so.2 time icinga2 daemon -C
+```
+
+Replace `/lib64/libjemalloc.so.2` with the absolute path
+you actually got from `ldconfig -p`!
+
+Please do us a favor and share your results
+[with us](https://community.icinga.com/c/icinga-2/6).
+
+### If it's faster with libjemalloc
+
+Run `systemctl edit icinga2.service`. This will open an editor.
+Add the following, save the file and close the editor.
+
+```
+[Service]
+Environment=LD_PRELOAD=/lib64/libjemalloc.so.2
+```
+
+Replace `/lib64/libjemalloc.so.2` with the absolute path
+you actually got from `ldconfig -p`!
+
+Restart Icinga and enjoy the speed.
+
 ## Required Information <a id="troubleshooting-information-required"></a>
 
 Please ensure to provide any detail which may help reproduce and understand your issue.
```

Comment:
Shouldn't be at the very top of the file. The following section clearly has an introductory character.